static/js/stat_var_hierarchy/stat_var_section.tsx

```diff
@@ -25,8 +25,12 @@ import React from "react";
 
 import { StatVarCharts } from "../browser/stat_var_charts";
 import { Context } from "../shared/context";
-import { NamedNode, StatVarHierarchyType } from "../shared/types";
-import { StatVarInfo, StatVarSummary } from "../shared/types";
+import {
```

Comment:
optional: Can't tell if changes in this package are related to the bar chart changes? If they are not, consider doing it in a separate PR.

Reply:
While not directly related, after adding `bar_tile.test.tsx`, the client side tests started failing on these test due to typing errors. Not totally sure why this started happening now, but it was blocking the PR submission